src/libmongoc/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ set (MONGOC_ENABLE_COMPRESSION_SNAPPY 0)
 set (MONGOC_ENABLE_COMPRESSION_ZLIB 0)
 set (MONGOC_ENABLE_COMPRESSION_ZSTD 0)
 
+set(MONGOC_OUTPUT_BASENAME "mongoc" CACHE STRING "Output mongoc library base name")
```

Comment:
Please add a space here -> `set (` for stylistic consistency

Reply:
done